src/app/component/Header/Header.container.js

```diff
@@ -359,7 +360,10 @@ export class HeaderContainer extends NavigationAbstractContainer {
             navigationState: { name }
         } = this.props;
 
-        if (isMobile.any() || name === CART_OVERLAY || (!isMobile.any() && name === SEARCH)) {
+        if (isMobile.any() 
```

Comment:
Please use concise language :')

Reply:
Please fix that manually, otherwise, it will show-up in every PR :)